go.mod

```diff
@@ -14,21 +16,21 @@ require (
 	github.com/aserto-dev/errors v0.0.5
 	github.com/aserto-dev/go-aserto v0.20.3
 	github.com/aserto-dev/go-authorizer v0.20.2
-	github.com/aserto-dev/go-directory v0.21.0
+	github.com/aserto-dev/go-directory v0.20.8-0.20230619123658-71cf8ee09ffc
```

Comment:
We may need to switch back to the proto version without the REST binding if we want to release this soon.
In a conversation with Gert and Omri we decided to introduce a v3 of the directory API with a few breaking changes to make certain optional inputs required and base the REST mapping on that v3 API in order to expose a more idiomatic API with GET, PUT, and DELETE requests on objects and relations.